apps/paf/translations/src/en/validation.json

```diff
@@ -313,7 +316,9 @@
     "after": "Enter a date after 01 01 1900"
   },
   "contact-number": {
-    "maxlength": "You can't use more than {{maxlength}} characters for your answer"
+    "maxlength": "You can't use more than {{maxlength}} characters for your answer",
+    "required": "Please enter a phone number",
```

Comment:
This required message is different to the others. They should probably all be the same unless theres a good reason.